lib/ansible/modules/cloud/openstack/os_stack.py

```diff
@@ -168,7 +168,10 @@ def _create_stack(module, stack, cloud, shade):
         else:
             module.fail_json(msg="Failure in creating stack: {0}".format(stack))
     except shade.OpenStackCloudException as e:
-        module.fail_json(msg=str(e))
+        if hasattr(e, 'response'):
+            module.fail_json(msg=str(e), response=e.response.json())
```

Comment:
Rather than use `str()` which will return a different type based on the Python version, use `to_native(e)` by importing `from ansible.module_utils._text import to_native`.

Reply:
Done. Usage of str(e) was already present there, but I have anyway replaced it with to_native(e). Thanks for hint